wdn/templates_4.1/scripts/main-wdn-plugins.js

```diff
@@ -1,8 +1,8 @@
 define([
 	'analytics',
+	'socialmediashare',
 	'navigation',
 	'search',
-	'socialmediashare',
```

Comment:
These are all async and order does not matter. If `navigation` depends on `socialmediashare` the latter should be in the former's dependency definition.